src/openrct2/world/Location.hpp

```diff
@@ -160,6 +160,11 @@ struct CoordsXY
     {
         return x == COORDS_NULL;
     };
+
+    void setNull()
```

Comment:
Our member functions are usually starting with a capital letter, any reason those null getter/setter are different?

Reply:
I noticed that too, the pair of `isNull/setNull` on `TileCoordsXY` use lowercase, so I just kept consistency here. I can do another PR later converting them all

Reply:
An oversight. I do think that renaming them in this PR would get a bit messy, so I'd prefer if it was done in another.